AdobeAnaltyicsAPI.gs

```diff
@@ -159,7 +260,65 @@ var AdobeAnalyticsAPIController = (function() {
 
   };
 
+  
+  /**
+   * AdobeAnalyticsAPIController.prototype.checkGlobalCompanyId()
+   * test your access token and get your global company IDs
+   *
+   * e.g. for curl: 
+   * curl -X GET --header "Accept: application/json" --header "x-api-key: {CLIENT ID}" --header "Authorization: Bearer {ACCESS_TOKEN}" "https://analytics.adobe.io/discovery/me"
+   * (see, https://www.adobe.io/apis/experiencecloud/analytics/docs.html#!AdobeDocs/analytics-2.0-apis/master/oauth-curl.md)
+   *
+   **/
+  AdobeAnalyticsAPIController.prototype.checkGlobalCompanyId = function() {
 
+    this.Logger("", {
+      "title": "checkGlobalCompanyId"
+    });
+
+    const headers = {
+      'Accecpt' : 'application/json',
+      'x-api-key' : API_KEY,
+      'Authorization' : 'Bearer ' + this.access_token
+    };
+
+    const payload = {};
+    
+    const options = {
+      'method': 'get',
+      'headers': headers
+    };
+
+    const url = "https://analytics.adobe.io/discovery/me";
+    
+    this.Logger(url, {
+      "prefix": "Request URL"
+    });
+    this.Logger(JSON.stringify(options), {
+      "prefix": "options"
+    });
+
+    const body = UrlFetchApp.fetch(url, options);
+    this.Logger(body, {
+      "prefix": "body"
+    });
+
+    var data = JSON.parse(body.getContentText("UTF-8"));
+
+    // save returned values to the sheet "Auth" (next to "Result of the function "checkGlobalCompanyId"")
```

Comment:
I think there's a lot of apostrophes.

Reply:
Thank you for your comment. I just removed the phrase in parentheses.